cartographer_grpc/framework/rpc.h

```diff
@@ -92,6 +100,15 @@ class Rpc {
   std::unique_ptr<::grpc::ServerAsyncReader<google::protobuf::Message,
                                             google::protobuf::Message>>
       server_async_reader_;
+  std::unique_ptr<::grpc::ServerAsyncReaderWriter<google::protobuf::Message,
+                                                  google::protobuf::Message>>
+      server_async_reader_writer_;
+
+  struct SendItem {
```

Comment:
style: declare types and structs right after private:

Reply:
Done.